docs/content/latest/api/ycql/ddl_create_index.md

```diff
@@ -107,6 +113,51 @@ When an index is created on an existing table, YugabyteDB will automatically bac
 
 - A unique index disallows duplicate values from being inserted into the indexed columns. It can be used to ensure uniqueness of index column values.
 
+### PARTIAL INDEX
+
+- If a `WHERE` clause is specified, only rows which satisfy the `index_predicate` are indexed.
+- An `index_predicate` can have sub-expressions on columns of these data types: `TINYINT`, `SMALLINT`, `INT/INTEGER`, `BIGINT`, `VARINT`, `BOOLEAN` and `TEXT` along with these operators (when applicable): `=, !=, >, <, >=, <=`.
+- Partial indexes can be `UNIQUE`. A UNIQUE partial index enforces the constraint that for each possible tuple of indexed columns, only one row that satisfies the `index_predicate` is allowed in the table.
+- `SELECT` queries can use a partial index for scanning if the `SELECT` statement's `where_expression` => (logically implies) `index_predicate`.
+
+{{< note title="Note" >}}
+
+- A partial index might not be chosen even if the implication holds in case there are better query plans.
+- The logical implication holds if all sub-expressions of the `index_predicate` are present as is in the `where_expression`.
+
+For example, assume `where_expression = A AND B AND C`, `index_predicate_1 = A AND B`, `index_predicate_2 = A AND B AND D`, `index_predicate_3 = A AND B AND C AND D`. Then `where_expression` only implies `index_predicate_1`
```

Comment:
nit: I think if we indent this with either 2 or 4 spaces (don't remember) then it will indent correctly as a part of the bullet above. Otherwise it will break the itemized list indentation.